<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateUserWindow</class>
 <widget class="QDialog" name="UpdateUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>Account Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_accountNumberError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_accountNumber">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>Full Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fullNameError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fullName">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ageError">
       <property name="styleSheet">
        <string notr="true">
QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_age"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_userNameError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_userName">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_passwordError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:500;

}</string>
       </property>
       <property name="text">
        <string>Confirm Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_confirmPasswordError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_confPassword">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_isAdmin">
     <property name="text">
      <string>Is Admin</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
